.github/workflows/docker-validate.yml

```
@@ -0,0 +1,192 @@
+name: Docker Validation
+
+on:
+  push:
+    branches: [ main, master ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)